jobs/future-effective-filings/k8s/job.param

```diff
@@ -0,0 +1,10 @@
+#=========================================================
+# Template parameters for:
+# Project: future-effective-filings
+# Component: job
+# Action: build
+# Template File: templates/job.json
+#=========================================================
+NAME="future-effective-filings"
+NAMESPACE="gl2uos"
+ENV="dev"
```

Comment:
Is this intended to run in Test? Should its filename reflect its environment?

Reply:
the param file values are just placeholders. I used the dev ones, but if you were going to build this in a differeent env you'd have to change them. This is consistent with all the param files I've made (except for ones with secrets)

Reply:
> Very cool, but honestly I don't understand how it all fits together. Could you put together a doc to explain how this is set up and calls its pieces? I think a quick tech transfer/demo to show the team how this works would be awesome too!

Haha yeah for sure I think this will need a demo. I did provide a doc in the other PR to entity that contains the integration pipeline as well, but I could add another here